kafka-scheduler/src/main/java/com/mesosphere/dcos/kafka/offer/PlacementRuleManager.java

```diff
@@ -0,0 +1,79 @@
+package com.mesosphere.dcos.kafka.offer;
+
+import com.mesosphere.dcos.kafka.config.KafkaSchedulerConfiguration;
+import com.mesosphere.dcos.kafka.state.FrameworkState;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.mesos.Protos;
+import org.apache.mesos.offer.constrain.*;
+import org.apache.zookeeper.Op;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Optional;
+
+class PlacementRuleManager {
+  private static final Log log = LogFactory.getLog(PlacementRuleManager.class);
+
+  private final FrameworkState frameworkState;
+
+  PlacementRuleManager(FrameworkState frameworkState) {
+    this.frameworkState = frameworkState;
+  }
+
+  public Optional<PlacementRule> getPlacementRule(
+          KafkaSchedulerConfiguration config,
+          Protos.TaskInfo taskInfo) {
+        Optional<PlacementRule> constraintRule = getPlacementConstraint(config);
```

Comment:
This is indented 4sp while the rest of the file is indented 2sp? Looks like the rest of Kafka is 2sp so it'd make sense to stick to that for now.